tiledb/sm/c_api/tiledb_experimental.h

```diff
@@ -519,6 +521,7 @@ TILEDB_EXPORT int32_t tiledb_subarray_add_point_ranges(
     const void* start,
     uint64_t count) TILEDB_NOEXCEPT;
 
+#ifndef TILEDB_REMOVE_DEPRECATIONS
```

Comment:
I think this might have been a mistake. It was committed deprecated??

Reply:
I guess it was added to make it usable to those that hadn't migrated to subarrays at that time.

Reply:
I believe this was an optimization we did for a customer. @ihnorton was the deprecated tag here a mistake?

Reply:
We're using the subarray version (`tiledb_subarray_add_point_ranges`) in TileDB-Py, so I think this is ok.